src/frontend/src/handler/explain.rs

```diff
@@ -65,63 +62,28 @@ async fn do_handle_explain(
                 cdc_table_info,
                 ..
             } => {
-                // TODO(st1page): refacor it
+                let col_id_gen = ColumnIdGenerator::new_initial();
+                // TODO(st1page): refactor it
```

Comment:
What and how to refactor? 🤣 These comments are so curt.

Reply:
This PR just moved the code around. cc. the author @st1page